### PR TITLE
Stac 11119 service name

### PR DESCRIPTION
**Pull request by hruhek**
### Step 1: Link to Jira issue
https://stackstate.atlassian.net/browse/STAC-11119

### Step 2: Description of changes
Sets right message title host and service event

### Step 3: Did you add / update tests for your changes in the right area?
- [x] Unit/Component test

### Step 4: I'm confident that everything is properly tested:
I got a PO / QA Approval by:
- [ ] Johnny

### Step 5: Can we ship this feature to production?
- [ ] Yes, I'm proud of my work. Ship it! :ship:

### CODE REVIEW

File: nagios/tests/test_nagios.py [hunk 1]
```diff
@@ -151,6 +144,71 @@ def test_create_event(self):
...
+                'type': 'CURRENT SERVICE STATE',
+                'parts': ['domU-12-31-38-00-78-98', 'Current Load', 'OK', 'HARD', '1', 'OK - load average: 0.04, 0.03'],
+                'msg_title': 'Current Load'
+            },
+            {
+                'type': 'SERVICE ALERT',
+                'parts': ['domU-12-31-39-02-ED-B2', 'cassandra JVM Heap', 'WARNING', 'SOFT', '1', ''],
+                'msg_title': 'cassandra JVM Heap'
+            },
+            {
```
Comment (TizianoPerrucci) [inline nagios/tests/test_nagios.py:168]: `PASSIVE SERVICE CHECK` event sample is missing

File: nagios/tests/test_nagios.py [hunk 2]
```diff
@@ -151,6 +144,71 @@ def test_create_event(self):
...
+                'type': 'SERVICE FLAPPING ALERT',
+                'parts': ['domU-12-31-39-16-52-37', 'cassandra JVM Heap', 'STARTED', 'Service started flapping'],
+                'msg_title': 'cassandra JVM Heap'
+            },
+            {
+                'type': 'ACKNOWLEDGE_SVC_PROBLEM',
+                'parts': ['domU-12-31-39-16-52-37', 'NTP', '2', '1', '0', 'nagiosadmin', 'alq'],
+                'msg_title': 'NTP'
+            },
+            {
```
Comment (TizianoPerrucci) [inline nagios/tests/test_nagios.py:188]: `ACKNOWLEDGE_HOST_PROBLEM` event sample is missing

File: nagios/stackstate_checks/nagios/nagios.py [hunk 3]
```diff
@@ -458,3 +433,32 @@ def _get_metric_prefix(self, line_data):
...
+
+
+def create_event(timestamp, event_type, hostname, fields):
+    """
+    Factory method called by the parsers
+    """
+    event = fields._asdict()
+    tags = ['{}:{}'.format(k, v) for k, v in event.items()]
+
+    # if host is localhost, turn that into the internal host name
```
Comment (TizianoPerrucci) [inline nagios/stackstate_checks/nagios/nagios.py:445]: any reason why you moved this above ?
before your change it was after the `event.update`

File: nagios/tests/test_nagios.py [hunk 4]
```diff
@@ -151,6 +144,71 @@ def test_create_event(self):
...
+                          'Service has entered a period of scheduled downtime'],
+                'msg_title': 'intake'
+            }
+        ]
+
+        for event in events:
+            self._check_event_msg_title(event_type=event['type'], parts=event['parts'], msg_title=event['msg_title'])
+
+    @staticmethod
+    def _check_event_msg_title(event_type, parts, msg_title):
```
Comment (TizianoPerrucci) [inline nagios/tests/test_nagios.py:205]: I would rename `msg_title` to `expected_msg_title`, will make it a bit more clear

**Review by TizianoPerrucci — CHANGES_REQUESTED** [review]

File: nagios/stackstate_checks/nagios/nagios.py (hunk 3 above)
Comment (hruhek) [reply nagios/stackstate_checks/nagios/nagios.py:445]: Good that you spotted that, no need for this change. 
I was testing one other approach in StackPack so I was sending a bit more data with tags. 
By mistake I didn't move the host part to its original line when I reverted the code.

File: nagios/tests/test_nagios.py (hunk 1 above)
Comment (hruhek) [reply nagios/tests/test_nagios.py:168]: Event sample added.

File: nagios/tests/test_nagios.py (hunk 2 above)
Comment (hruhek) [reply nagios/tests/test_nagios.py:188]: Event sample added.

**Review by TizianoPerrucci — APPROVED** [review]

**Pull request merged**